app/styles/_data-toolbar.less

```diff
@@ -1,6 +1,12 @@
 /* Data toolbar
    - Includes customizations for label filter input and active filter
 ---------------------------------------------------------------------------- */
+@data-toolbar-filter-max-width:   772px; // 832 - 60; container - (left + right margin) at 992px viewport width
```

Comment:
Thanks for making these variables.

Unclear where the 832 comes from. Would be better to calculate from variables like `@grid-gutter-width` and `@screen-md` etc where possible.

Same comment for some of the magic numbers below.